bin/lib/mailer.js

```diff
@@ -125,6 +125,60 @@ The audio management page is <a href="${data.managementURL}">here</a>.
 	;
 }
 
+function sendCustomMessageToSpecifiedRecipients(recipients = [], subject, plainText, htmlContent){
+
+	if(!recipients || recipients.length < 1){
+		return Promise.reject('No recipients passed. Message not sent');
+	}
+
+	const post_body_data = {
+		transmissionHeader: {
+			description: 'Custom FT Labs Absorber email',
+		    metadata: {
+		        audioArticleIngestionUuid: 'custom-message'
+		    },
+		},
+		to: {
+		    address: recipients
+		},
+		from: {
+		    address: from_email_address,
+		    name:    from_email_name
+		},
+		subject:          subject,
+		htmlContent:      plainText,
+		plainTextContent: htmlContent
+	};
+
+	return fetch(mail_post_url, {
+			method       : 'POST', 
+			body         :  JSON.stringify(post_body_data),
+			headers      : {
+				'Content-Type'  : 'application/json',
+				'Authorization' : mail_post_auth_token
+			}
+		})
+		.then(res => {
+			if(!res.ok){
+				throw res;
+			} else {
+				return res;
+			}
+		})
+		.then(res => res.text())
+		.then(response => debug(response))
+		.then(function(){
+			return true;
+		})
+		.catch(err => {
+			debug('An error occurred sending a custom email', err);
+			return false;
+		})
+	;
+
+}
+
 module.exports = {
-	send : sendMessage
+	send : sendMessage,
```

Comment:
why not rename this to make it clearer it is a specific kind of message, i.e. announcing the arrival of a new audio file, or replace the one invocation of it with a call to  sendCustomMessage instead?